### PR TITLE
chore(ci): e2e signatures tests

### PR DESCRIPTION
**Pull request by danielpacak**
Run signatures tests as sanity check on each PR.

### CODE REVIEW

**Review by rafaeldtinoco — APPROVED** [review]
LGTM. Following TODOs I see: I really see a need for us to move Dockerfile and Makefile around with better namings for the tests. Or, at least, have them renamed to something else. I think we can do this soon, not needed in this PR. Just sharing a feeling.

File: tests/tracee_test.go [hunk 1]
```diff
@@ -188,9 +188,21 @@ func TestTraceeSignatures(t *testing.T) {
...
-		for _, sigid := range []string{"TRC-3", "TRC-4", "TRC-9", "TRC-10", "TRC-11"} {
-			t.Run(fmt.Sprintf("%s/%s", image, sigid), func(t *testing.T) {
+	// Ubuntu 20.04 provided by GitHub Actions runner does not support CO:RE.
+	// Thus, we are running end-to-end signatures tests using tracee non CO:RE
+	// container image.
+
+	// FIXME Pass tracee container image flavor (tracee-nocore, tracee-core, etc.)
+	//       as input parameter to this test so we can set in the CI workflow
+	//       instead of hardcoding it here. The actual logic of the test should be
+	//       agnostic of tracee container flavor.
```
Comment (grantseltzer) [inline tests/tracee_test.go:198]: Can we have an issue for this instead of a FIXME comment?

File: tests/tracee_test.go [hunk 2]
```diff
@@ -188,9 +188,21 @@ func TestTraceeSignatures(t *testing.T) {
...
+	for _, image := range []string{"tracee-nocore"} {
+		// FIXME Pass signature identifiers (TRC-3, TRC-4, TRC-9, etc.) as input
+		//       parameter to this test so we can use it as smoke test in the
+		//       PR validation workflow (with TRC-2) only or as full-blown end-to-end
+		//       nightly test run.
```
Comment (grantseltzer) [inline tests/tracee_test.go:203]: Same here

File: tests/tracee_test.go (hunk 1 above)
Comment (danielpacak) [reply tests/tracee_test.go:198]: I believe that to close #1191 and #1234 we must resolve these comments. And I'm actually going to jump right on it after we merge this PR.

File: tests/Makefile [hunk 3]
```diff
@@ -1,14 +1,2 @@
...
-	cd .. && make -f builder/Makefile.tracee-container build-alpine-tracee-nocore
-
-build-tracee-docker-core-btfhub:
-	cd .. && make -f builder/Makefile.tracee-container build-alpine-tracee-core-btfhub
-
-run-tests: build-trainer-docker build-tracee-docker-core build-tracee-docker-nocore build-tracee-docker-core-btfhub
-	go test -v -run=TestTraceeSignatures
\ No newline at end of file
+docker-build-tracee-trainer:
+	docker build -t tracee-trainer:latest tests
```
Comment (simar7) [inline tests/Makefile:2]: Is this correct? 
```vagrant@ubuntu-impish:~/repos/tracee/tests$ make docker-build-tracee-trainer 
docker build -t tracee-trainer:latest tests
unable to prepare context: path "tests" not found
make: *** [Makefile:2: docker-build-tracee-trainer] Error 1
```

Maybe you meant: `docker build -t tracee-trainer:latest .`?

Comment (simar7) [reply tests/Makefile:2]: Sorry ignore that - I understood your motivation now and where it would be run from.

Comment (danielpacak) [reply tests/Makefile:2]: It's a temporary change to run it from the root directory. Relative paths are tricky. We are going to revisit makefile targets with @rafaeldtinoco to make it consistent with other build scripts

File: tests/tracee_test.go (hunk 1 above)
Comment (danielpacak) [reply tests/tracee_test.go:198]: See #1411 that resolves FIXME and TODOs

**Pull request merged**